backend/clickhouse/log_row.go

```diff
@@ -86,6 +86,15 @@ func WithSeverityText(severityText string) LogRowOption {
 	}
 }
 
+func WithSource(source string) LogRowOption {
+	return func(h *LogRow) {
+		if source == highlight.SourceAttributeFrontend {
+			source = "frontend"
+		}
+		h.LogAttributes[highlight.SourceAttribute] = source
```

Comment:
What do you think about writing this to `ServiceName` instead? We'd have to do some additional wiring up (which I'm happy to handle) but I think it would be a lot more optimized to use a top level column over adding more things to the log attributes map.

Reply:
makes sense, especially when the otel semconv `ServiceName` attribute is not set (i'll also check that when it is, we pipe it thru to this top-level column). what do I need to change on the frontend to wire this?

Reply:
I think adding a new column makes a lot of sense! Let's do that.

`ServiceName` - `highlight-js-sdk`
`Source` - `frontend`

But long term, we could have `Source` be something the user could actually customize. Would be really helpful if they had multiple frontend apps. 

cc @ccschmitz - per our convo last week about having a way to name sources in the setup page.

Reply:
updated!